services/mirror/mirror_pull.go

```diff
@@ -613,14 +613,8 @@ func checkAndUpdateEmptyRepository(ctx context.Context, m *repo_model.Mirror, re
 		}
 		// Update the git repository default branch
 		if err := gitrepo.SetDefaultBranch(ctx, m.Repo, m.Repo.DefaultBranch); err != nil {
-			if !git.IsErrUnsupportedVersion(err) {
-				log.Error("Failed to update default branch of underlying git repository %-v. Error: %v", m.Repo, err)
-				desc := fmt.Sprintf("Failed to update default branch of underlying git repository '%s': %v", m.Repo.RepoPath(), err)
-				if err = system_model.CreateRepositoryNotice(desc); err != nil {
-					log.Error("CreateRepositoryNotice: %v", err)
-				}
-				return false
-			}
+			log.Error("Failed to update default branch of underlying git repository %-v. Error: %v", m.Repo, err)
+			return false
```

Comment:
Maybe we should keep the notice here because the mirror is run in the backend.

Reply:
Who cares about the system notice? Who really ever used it?

Reply:
Actually, I will and some reports come from the system notice.

Reply:
Have you ever seen "Failed to update default branch of underlying git repository '%s': %v" ? And what would you do if you see it?

Reply:
Never. :)